### PR TITLE
chore: konflux-related jobs should use konflux cp

### PR DESCRIPTION
**Pull request by psturc**
[KONFLUX-4343](https://issues.redhat.com//browse/KONFLUX-4343)

### CODE REVIEW

File: clusters/hosted-mgmt/hive/pools/rhtap/rhtap-ocp-4-15-0-amd64-aws-us-east-2_clusterpool.yaml
Comment (psturc) [inline clusters/hosted-mgmt/hive/pools/rhtap/rhtap-ocp-4-15-0-amd64-aws-us-east-2_clusterpool.yaml:36]: after merging this PR there will be still some rhtap jobs using this cluster pool:
* [rhtap-cli](https://github.com/openshift/release/blob/591ef65bac1ba3d5d1ab1d77d0053ac968b64f90/ci-operator/config/redhat-appstudio/rhtap-cli/redhat-appstudio-rhtap-cli-main.yaml#L48-L114)
* [tssc](https://github.com/openshift/release/blob/591ef65bac1ba3d5d1ab1d77d0053ac968b64f90/ci-operator/config/redhat-appstudio/tssc-sample-pipelines/redhat-appstudio-tssc-sample-pipelines-main.yaml#L81-L111)

so I reduced the size to half for now

**Comment by psturc** [comment]
/assign @dirgim @mmalina @dperaza4dustbit @johnmcollier @vibe13 @jgwest

Hello approvers! I will need an approval (or `/approve` comment) from you in order to merge this one.

Thank you!

**Comment by sadlerap** [comment]
/approve

Changes for konflux-workspaces are fine by me.

**Review by mmalina — APPROVED** [review]
/approve

**Review by kasemAlem — APPROVED** [review]

File: ci-operator/config/konflux-ci/release-service-catalog/konflux-ci-release-service-catalog-development.yaml
Comment (jinqi7) [inline ci-operator/config/konflux-ci/release-service-catalog/konflux-ci-release-service-catalog-development.yaml:25]: Do we need move it to "us-east-2" if  no more test will use "us-west-2" cluster pool?

File: ci-operator/config/konflux-ci/release-service/konflux-ci-release-service-main.yaml
Comment (jinqi7) [inline ci-operator/config/konflux-ci/release-service/konflux-ci-release-service-main.yaml:33]: And this one?

File: clusters/hosted-mgmt/hive/pools/rhtap/rhtap-ocp-4-15-0-amd64-aws-us-east-2_clusterpool.yaml
Comment (psturc) [reply clusters/hosted-mgmt/hive/pools/rhtap/rhtap-ocp-4-15-0-amd64-aws-us-east-2_clusterpool.yaml]: after follow-up discussion we decided to scale down all "rhtap" cluster pools to zero and let all konflux and rhtap-related jobs use only the "konflux" cluster pool

**Pull request merged**
